docs/content/docs/custom-resource/reference.md

```diff
@@ -67,7 +67,7 @@ This page serves as a full reference for FlinkDeployment custom resource definit
 | Parameter | Type | Docs |
 | ----------| ---- | ---- |
 | job | org.apache.flink.kubernetes.operator.api.spec.JobSpec | Job specification for application deployments/session job. Null for session clusters. |
-| restartNonce | java.lang.Long | Nonce used to manually trigger restart for the cluster/session job. In order to trigger  restart, change the number to anything other than the current value. |
+| restartNonce | java.lang.Long | Nonce used to manually trigger restart for the cluster/session job. In order to trigger restart, change the number to anything other than the current value. |
```

Comment:
The reference page is generated based on JavaDocs, you need to fix it there, you cannot edit this page directly

Reply:
> The reference page is generated based on JavaDocs, you need to fix it there, you cannot edit this page directly

Okay, Thanks for your review ,I got it, and will update it.

Reply:
@gyfora It is a space caused by a line break in the doc. Is there any solution for this? Thank you!

Reply:
The solution in that case would be improving the doc generator to automatically remove double spaces.

Reply:
Okay ,I got it.

Reply:
@gyfora Please take a look again, thanks!